src/api/middleware.js

```diff
@@ -7,5 +7,26 @@ const getMiddlewareApi = () => axios.create({
 export const getComplimentaryLLD = async (usingWalletAddress) => {
   const userToken = JSON.parse(localStorage.getItem('ROCP_token'));
   const middlewareApi = getMiddlewareApi();
-  return middlewareApi.post('/v1/onboarding', { usingWalletAddress, userToken }).catch((e) => e.response);
+  return middlewareApi
+    .post('/v1/onboarding', { usingWalletAddress, userToken })
+    .catch((e) => e.response);
+};
+
+export const generatePdf = async (usingWalletAddress, companyId, pathName, blockNumber) => {
+  const userToken = JSON.parse(localStorage.getItem('ROCP_token'));
+  const middlewareApi = getMiddlewareApi();
+  try {
+    const response = await middlewareApi.post('/v1/certificate', {
+      usingWalletAddress,
+      userToken,
+      companyId,
+      pathName,
+      blockNumber,
```

Comment:
you dont need user token in fact middleware only uses

 	const { companyId, pathName, blockNumber } = req.body;